header.php

```diff
@@ -48,24 +48,29 @@
 </svg>
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php esc_html_e( 'Skip to content', 'pressbooks-aldine' ); ?></a>
-	<header class="header" role="banner" style="background-image: url(
 	<?php
 
-	if ( has_post_thumbnail() ) {
-		echo get_the_post_thumbnail_url();
+	$template = get_page_template_slug();
```

Comment:
I feel this become very nasty what if we make this more succinct? what do you think?

```php
<?php
$template = get_page_template_slug();
$template_dir = get_template_directory_uri();

if ($template === 'page-featured-books.php') {
    echo "<header class='header' role='banner'>";
    if (has_post_thumbnail()) {
        echo get_the_post_thumbnail_url();
    }
    return;
}

$bg_image = '';

if (is_front_page()) {
    $bg_image = has_header_image() ? get_header_image() : $template_dir . '/dist/images/header.jpg';
} else {
    $bg_image = $template_dir . '/dist/images/catalog-header.jpg';
}

echo "<header class='header' role='banner' style='background-image: url({$bg_image})'>";
?>
```